### PR TITLE
Few minor fixes

### PR DESCRIPTION
**Pull request by microadam**
Mainly fixes issue #107 where deleting a record in admin causes pagination to reset

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ function log() {
 // bundles need all of these.
 serviceLocator
   .register('properties', properties)
-  .register('logger', { info: log, debug: log, warn: log, error: log })
+  .register('logger', { info: log, debug: log, warn: log, error: log, verbose: log, silly: log })
```
Comment (serby) [inline app.js]: Please remove verbose, I've deliberately refactored it out. Try updating your packages if you are having problems.

**Comment by serby** [comment]
Did you get muy message about updating your npm modules

**Comment by serby** [comment]
Are you getting these messages?

**Comment by microadam** [comment]
Yeah, sorry. Haven't had a chance to look at it again. Will try and fix it
today

On 22 Nov 2012, at 09:15, Paul Serby notifications@github.com wrote:

Are you getting these messages?

—
Reply to this email directly or view it on
GitHubhttps://github.com/serby/ctrl/pull/138#issuecomment-10627888.

**Comment by microadam** [comment]
Updated npm modules and it works now without the verbose

**Comment by serby** [comment]
silly also needs removing

**Comment by serby** [comment]
Can you remove silly, with QA, so I can pull this please.

**Comment by microadam** [comment]
Have made the change

**Pull request merged**